src/app/components/Curation/HierarchicalGrid/index.tsx

```diff
@@ -60,6 +60,9 @@ const HiearchicalGrid = ({
 
           const lazyLoadImages = !(isFirstPromo && isFirstCuration);
 
+          const fetchPriorityImages =
```

Comment:
Use the same name as the attribute?
```suggestion
          const fetchpriority =
```